snforge_std/src/trace.cairo

```diff
@@ -0,0 +1,60 @@
+use starknet::testing::cheatcode;
+use starknet::ContractAddress;
+use super::PrintTrait;
+
+#[derive(Drop, Serde, PartialEq)]
+struct CallEntryPoint {
+    entry_point_type: EntryPointType,
+    entry_point_selector: felt252,
+    calldata: Array<felt252>,
+    storage_address: ContractAddress,
```

Comment:
What the storage_address is? 🤔

Reply:
I can rename it, the pathfinder rpc node names it `contract_address`, is that fine for you?